src/cmd/cloud.js

```diff
@@ -54,7 +54,7 @@ class CloudCommand {
 		return api.claimDevice(deviceId).then(() => {
 			console.log('Successfully claimed device ' + deviceId);
 		}, (err) => {
-			if (err && typeof err === 'string' && err.indexOf('That belongs to someone else.') >= 0) {
+			if (err && typeof err[0] === 'string' && err[0].indexOf('That belongs to someone else.') >= 0) {
```

Comment:
how about adding a test so we don't hit the same issue down the road?

https://github.com/particle-iot/particle-cli/blob/master/test/e2e/cloud.e2e.js#L174-L185

Reply:
Could you help me with that since the test needs to answer a prompt. It's not just checking stdout. It will also need a new environment variable for a device that is claimed to another account.